jsonGeneratorTests.js

```diff
@@ -46,4 +46,16 @@ describe('JSONGenerator Tests', function() {
 
         generateResult.should.eql(resp)
     })
+
+    it('Valid JSON - FROM, COMMENT', async function() {
+
+        let buf = Buffer.from('FROM nginx:latest\n# Some Value\n');
+        let bufferStream = new Stream.PassThrough();
+        bufferStream.end(buf);
+
+        let generateResult = await jsonGenerator.generateJSON(bufferStream)
+
+        let expectedValue = 'Some Value'
+        generateResult[Object.keys(generateResult)[1]].should.eql(expectedValue)
```

Comment:
Since the comments are generated with a random suffix, I wasn't sure the best way to test the result besides this -- open to suggestions for how to handle either testing or multiple comments.

Reply:
We can represent the comment as embedded Object.  This object contains a value property which is the text part of the comment and also contains a type property, what refers the object is a type. 

For example: 
`
{ value: 'My comment', type: 'comment' }
` 

What do you think about it?